heartbeat/nfsserver

```diff
@@ -269,7 +301,7 @@ nfs_exec()
 	set_exec_mode
 
 	case $EXEC_MODE in 
-		1) ${OCF_RESKEY_nfs_init_script} $cmd;;
+		1) unshare -u /bin/sh -c "hostname ${OCF_RESKEY_nfs_server_scope}; exec ${OCF_RESKEY_nfs_init_script} $cmd" ;;
```

Comment:
You should check if nfs_server_scope is set before doing unshare.

Reply:
Right. Perhaps use a function which would print the scope to stdout. The function could then perhaps try to use a different cluster-specific string as the scope; iirc, we discussed something along those lines.

Reply:
right, I'll add the "original" in the unset branch.
I decided against "making up" a scope name, it's just too much implicit magic (that needs to be correct and maintained), compared to just setting it explicitly, and pointing it out in documentation.